recipes/finmind/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "finmind" %}
+{% set version = "1.0.84" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/FinMind-{{ version }}.tar.gz
+  sha256: f19bc2a7c9698ea80c4a78d2d3eb27daad01c8bec896cc34da9ff8f74a413644
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - FinMind
+
+
+about:
+  home: https://github.com/FinMind/FinMind
+  license: MIT
```

Comment:
The license here is ``MIT``, however, you packaged ``Apache-2.0`` (and it is also missing some information in the license file)